homeassistant/helpers/storage.py

```diff
@@ -80,11 +80,10 @@ def path(self):
             data = self._data
         else:
             data = await self.hass.async_add_executor_job(
-                json.load_json, self.path, None)
+                json.load_json, self.path, [])
```

Comment:
This is not technically the same. If we would store an empty list or dictionary in JSON, we would now treat it as no config file being available.

Reply:
Changed to `{}` which is the default value.

Any config is expected to have `version` attribute in the line below.

Reply:
Good point